src/toM3u8.js

```diff
@@ -246,10 +246,11 @@ export const organizeAudioPlaylists = (playlists, sidxMapping = {}, isAudioOnly
 export const organizeVttPlaylists = (playlists, sidxMapping = {}) => {
   return playlists.reduce((a, playlist) => {
     const label = playlist.attributes.label || playlist.attributes.lang || 'text';
+    const language = playlist.attributes.lang || playlist.attributes.label || 'en';
```

Comment:
It's more correct to default to `und` rather than `en` because we don't want to mark something as `en` when it isn't.

From dash spec 5.3.3.2 and IETF RFC 5646 section 4.1 step 5 https://datatracker.ietf.org/doc/html/rfc5646#section-4.1

Reply:
Actually, should we even bother trying to set it to label? We're more likely to have a language set than a label based on mpd files I've seen. So, might be better to do `playlist.attributes.lang || 'und'`

Reply:
Thanks for the documentation! Just made that change so we fallback to `und`

Reply:
you're welcome!